README.md

```diff
@@ -305,21 +87,33 @@ Jib packages your Java application into the following paths on the image:
 
 Running commands like `apt-get` slows down the container build process. We **do not recommend or support** running commands as part of the build. 
 
-However, if you need to run commands, you can build a custom base image. You can then use this custom base image in the `jib-maven-plugin` by adding the following configuration:
+However, if you need to run commands, you can build a custom base image. 
```

Comment:
Maybe this question can be split up into the respective maven and gradle readmes.

Reply:
Oh wait, there already are `from` examples. Are the custom base image examples here really necessary then?

Reply:
Not completely necessary, but just gives a direct pointer for readers looking for the specific solution.

Reply:
It might be better to keep the FAQ concise though so people can easily scan for what they want.

You could be more specific without examples too, like changing the last sentence to "However, if you need to run commands, you can build a custom base image **and use it with the `from` configuration parameter**."

Reply:
I put it into a collapsible section so that it doesn't take up any extra space.

Reply:
The FAQ will probably be moved to a Wiki page or something rather than remain on the README.